core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -769,7 +769,6 @@ private[spark] object Utils
    * logic of locating the local directories according to deployment mode.
    */
   def getConfiguredLocalDirs(conf: SparkConf): Array[String] = {
-    val shuffleServiceEnabled = conf.get(config.SHUFFLE_SERVICE_ENABLED)
```

Comment:
To be clear, this is enough for a valid PR. We can merge if we have only this removal.

Reply:
done